comdb2/dbapi2.py

```diff
@@ -240,24 +240,52 @@
    cursor for that connection.
 """
 
+from __future__ import annotations
+
 import functools
 import itertools
 import weakref
 import datetime
 import re
 
 from . import cdb2
-
-__all__ = ['apilevel', 'threadsafety', 'paramstyle',
-           'connect', 'Connection', 'Cursor',
-           'STRING', 'BINARY', 'NUMBER', 'DATETIME', 'ROWID',
-           'Datetime', 'DatetimeUs', 'Binary', 'Timestamp', 'TimestampUs',
-           'DatetimeFromTicks', 'DatetimeUsFromTicks', 'TimestampFromTicks',
-           'Error', 'Warning', 'InterfaceError', 'DatabaseError',
-           'InternalError', 'OperationalError', 'ProgrammingError',
-           'IntegrityError', 'DataError', 'NotSupportedError',
-           'UniqueKeyConstraintError', 'ForeignKeyConstraintError',
-           'NonNullConstraintError']
+from .cdb2 import Row, Value
+from typing import Any, Callable, Iterator, Mapping, Sequence
```

Comment:
Same as above, it may be better to update some of these to come from `collections.abc`.